src/core/vdom.ts

```diff
@@ -1296,7 +1335,7 @@ export function renderer(renderer: () => RenderResult): Renderer {
 			};
 		}
 
-		domNode.addEventListener(eventName, callback);
+		domNode.addEventListener(eventName, callback, options);
```

Comment:
I think you will have to do a check here to support IE11, as it doesn't take an options bag so passing anything truthy to it will mean it does a capture.

Reply:
So IE doesn't seem to have an equivalent here, could we still implement this with a note saying IE doesn't support event options?

Reply:
Yeah none of the options bag is implemented in IE other than capture and it’s a Boolean as the third parameter I believe.

Reply:
Yup that’s correct, it doesn’t error passing an object bag but it does consider it “truthy” so capture is enabled. We’ll need to sniff IE11 and and not pass the options bag at all I think. That can be documented that the “passive” helper isn’t supported in IE11